Makefile

```diff
@@ -1,10 +1,24 @@
 #!make
 
+SHELL := /bin/bash
+
 fmt:
-	pre-commit run --all-files
+	@pre-commit run --all-files
 
 fmt-unstaged:
-	pre-commit run --files $(shell git diff --name-only)
+	@pre-commit run --files $(shell git diff --name-only)
 
 test:
-	pytest -n auto --maxfail=3
\ No newline at end of file
+	@pytest -n auto --maxfail=3
+
+pip-compile:
+	@pip-compile --resolver=backtracking requirements.in --generate-hashes
+	@pip-compile --resolver=backtracking dev-requirements.in --generate-hashes
+
+py-install:
+	@if [ -d ".venv" ]; then source .venv/bin/activate && deactivate && rm -rf .venv; fi
+	@python -m venv .venv
+	@source .venv/bin/activate
+	@pip install -r requirements.txt -r dev-requirements.txt
+
+py-upgrade: pip-compile py-install
```

Comment:
On peut p'tet se passer de cette commande et faire un `make freeze install` ? Le seul cas que je vois c'est quand on update un .in et qu'on veut update son env de dev

Reply:
ok!

Reply:
Du coup j'ai enlevé la commande, ça n'a plus trop de sens je pense si c'est juste pour appeler `pip install`